report.tex

```diff
@@ -155,24 +155,23 @@ \subsubsection{The Scenario Page}
 \end{figure*}
 The accompanying scenario page shown in Figure \ref{fig2} will give the user the options to input all necessary data and efficiently view all of their options. This will provide a visual interface for the user to modify their scene without added interaction.
 
-Additionally, there is the option to outline the elements in red or some other colour to indicate a non-conforming piece of data. In the tab like display, the user is not aware of the exact location of flaws in their data.
+Additionally, there is the option to outline the elements in red or some other colour to indicate a non-conforming piece of data.
 
 We could also implement a dropdown in the sidebar in each scene tat gives a short summary of the data entered and can also indicate errors in the data entry. This will give users an even more general view of their scene.
 
 \textbf{Note}: We are unsure of the difference between the emotion and reaction elements. The two descriptions seem to be linked, and as such we wonder if the program could eliminate one or the other.
 
 \subsection{Functionality}
-With the implementation of a sidebar navigation paradigm, we can eliminate the need for a lot of the confusing actions that were present in the tab layout. This includes the arrows above the tabs and the delete button (which could now be placed on the sidebar element or in its dropdown menu), as well as a myriad of alignment problems. Now the data has one single point of control that can be stored as one object for manipulation and viewing.
+
+Introducing a sidebar navigation paradigm can streamline the user experience by simplifying actions previously executed through the tab layout. This includes eliminating arrows above tabs and the delete button, which can now be integrated into the sidebar element or its dropdown menu. Additionally, this paradigm resolves various alignment issues, consolidating data control into a single point for easier manipulation and viewing.
 
 \subsubsection{Saving}
-We suggest that the 'Scenarios' folder hold all of the user's stored scenarios. The user's account status and data could be put on a ribbon across the top of the screen, potentially containing links to the other tools Illumia Labs licenses.
 
-This would not only allow users to store their previously constructed scenarios and use template ideas, but also encourage the use of the entire ecosystem developed by Illumia Labs.
+We propose organizing all user-stored scenarios within a dedicated 'Scenarios' folder. Account status and data could be displayed on a ribbon across the screen's top, potentially featuring links to other tools licensed by Illumia Labs. This approach not only facilitates scenario storage and template usage but also promotes engagement with Illumia Labs' entire ecosystem.
```

Comment:
"... scenario storage and template usage, but also promotes..."
comma added for punctuation and sentence structure, again not necessary.

Reply:
done